python/lsst/ctrl/bps/wms/panda/idds_tasks.py

```diff
@@ -237,24 +242,3 @@ def get_input_file(self, job_name):
         quantum graph file name
         """
         return next(iter(self.bps_workflow.nodes.get(job_name).get("inputs")))

```

Comment:
Another heads up, this is one of the lines that won't work after the execution butler changes because it isn't using the API and the internal details are changing.  (self.bps_workflow.get_job_inputs(job_name, data=True)  Not sure if you need just the GenericWorkflowFile names or the whole objects.